lib/Language/en/translation.json

```
@@ -1433,5 +1431,29 @@
   },
   "term": {
     "missingContent": "missing content"
+  },
+  "itemSearchTool": {
+    "title": "Search {{itemName}}",
+    "loading": "Loading search parameters",
+    "loadError": "Error loading search parameters. Check console for detailed errors.",
+    "noParameters": "No searchable parameters found",
+    "searching": "Searching",
+    "searchError": "Search failed. Check console for detailed errors.",
+    "numericParameter": {
+      "minimum": "Minimum",
+      "maximum": "Maximum"
+    },
+    "searchBtnText": "Search",
+    "resetBtnText": "Clear input",
+    "backBtnText": "Search again",
+    "resultsCount": {
+      "zero": "No match found",
+      "one": "Showing 1 match",
+      "many": "Showing {{count}} matches"
+    }
```

this probably should use i18next pluralization option, as described in #5113
```suggestion
    "resultsCount": {
      "zero": "No match found",
      "count": "Showing {{count}} match",
      "count_plural": "Showing {{count}} matches"
    }
```

---

Thanks for the suggestion!

I have simplified it a bit further by removing the translation for `zero` so we just have `resultsCount` and `resultsCount_plural`.